packages/cli/src/tools/downloader.ts

```diff
@@ -0,0 +1,34 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+
+import fetch from 'node-fetch';
+
+/**
+ * Downloads the contents of the given `url`
+ * into the temp folder of the OS.
+ */
+const downloader = (url: string): Promise<string> => {
```

Comment:
`downloader` is not a good name for this utility. I'd go with something like `downloadToTmpdir`. Also, maybe it's worth to remove the downloader once we're done or failed to do stuff?

Reply:
Moved to `fetch.ts`. Not sure what you mean by remove the `downloader` 🤔